src/main/java/pacApp/pacModel/Currency.java

```diff
@@ -0,0 +1,13 @@
+package pacApp.pacModel;
+
+public enum Currency {
+    USD, EUR;
```

Comment:
Why does the backend know USD as well as EUR? Why not more currenies, or just USD?

Reply:
I thought the currencies are needed for an automatic conversion of the car and rental costs based on the default currency of the user for the frontend. Nevertheless, additional currencies are needed to be added. Please correct me @cristianCiu.